docs/_static/auth/iam.yaml

```diff
@@ -406,6 +406,42 @@ paths:
           description: |
             This is returned when an OAuth2 Client is unauthorized to talk to the introspection endpoint.
             Note: introspection of an invalid or malformed token returns a 200 where with field 'active'=false
+  /iam/{id}/status/{page}:
```

Comment:
this is a really generic path, as we also have `/authorize/`, `/callback` etc (maybe those should include `/oauth`, but out of scope). Maybe make it `/credential/status`?